.eslintrc.js

```diff
@@ -119,10 +120,17 @@ module.exports = {
       errorIfStrict,
       { some: ["nesting", "id"] },
     ],
+    "no-console": OFF,
+    "no-shadow": OFF,
+    "@typescript-eslint/no-shadow": ERROR,
     // disallow use of undeclared variables unless mentioned in a
     // /*global */ block
-    "no-undef": ERROR,
-    "no-use-before-define": [ERROR, { functions: false, variables: false }],
+    "no-undef": OFF,
+    "no-use-before-define": OFF,
+    "@typescript-eslint/no-use-before-define": [
```

Comment:
when should we use rules `@typescript-eslint` as opposed to regular eslint?

Reply:
I think if a typescript version of a rule is available it should be used since those would take precedence. They usually do everything the original version of the rule does and add typescript support